ecodse_funtime_alpha/train.py

```diff
@@ -16,24 +19,62 @@ def train_loop(dataset, model, optimizer):
         print(loss)
 
 
-def fit_loop(dataset, model, optimizer):
-    dataset = dataset.shuffle(12)
-    dataset = dataset.repeat(20)
-    dataset = dataset.batch(4)
+def fit_loop(dataset, model, optimizer, nepoch, batchsize):
+    nstep = len(list(dataset)) // batchsize
+    dataset = dataset.shuffle(12)  # 12 = buffer size
+    dataset = dataset.repeat(nepoch)
+    dataset = dataset.batch(batchsize)
     model.compile(optimizer=optimizer,
                   loss=tf.keras.losses.binary_crossentropy,
                   metrics=["accuracy"])
-    model.fit(dataset, epochs=20, steps_per_epoch=2)
+    model.fit(dataset, epochs=nepoch, steps_per_epoch=nstep)
+
+
+def get_args(args):
+    argparser = argparse.ArgumentParser()
+    argparser.add_argument('--imagepath',
+                           default='../../rainforest/fixed-train-jpg/',
+                           help='path to image directory')
+    argparser.add_argument('--labelpath',
+                           default='../../rainforest/train_v3.csv',
+                           help='path to csv file for labels')
+    argparser.add_argument('--seed',
+                           default=-1,
+                           type=int,
+                           help='Set random seed to this number (if >0)')
+    argparser.add_argument('--kernels',
+                           default=1,
+                           type=int,
+                           help='Number of kernels in the CNN')
+    argparser.add_argument('--ksize',
+                           default=1,
+                           type=int,
+                           help='Size of kernels in CNN')
+    argparser.add_argument('--lr',
+                           default=0.1,
+                           type=float,
+                           help='Learning rate (default=0.1')
```

Comment:
You should be consistent for putting the default in the help string or not.

If you do you should use a variable to be sure the value is always up to date.
```
default_lr = 0.1
...
...
...
argparser.add_argument('--lr',
                           default=default_lr,
                           type=float,
                           help=f"Learning rate (default={default_lr}")
```

Reply:
ok, solved